README.md

```diff
@@ -13,6 +13,8 @@ Arguments:
       --client-id string       API Client ID (env:SOPHOS_CLIENT_ID)
       --client-secret string   API Client Secret (env:SOPHOS_CLIENT_SECRET)
       --show-all               List all non-ok endpoints
+      --page-size uint32       Amount of objects to fetch during each API call (default 100)
+      --exclude stringArray    Alerts to ignore. Can be used multiple times and supports regex.
```

Comment:
Note to self: Should maybe be `--exclude-alert` so that we can also have `--exclude-endpoint`

Reply:
Done